### PR TITLE
Aggregate summary

### PR DESCRIPTION
**Pull request by Gustry**

### CODE REVIEW

File: safe/definitionsv4/fields.py [hunk 1]
```diff
@@ -482,6 +525,7 @@
     adult_count_field,
     elderly_count_field,
     size_field,
+    total_field,
```
Comment (ismailsunni) [inline safe/definitionsv4/fields.py]: Why does total go to exposure_fields? Should it go to aggregate_hazard layer?

File: safe/gisv4/vector/aggregate_summary.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        should accept params 'current' (int), 'maximum' (int) and 'step' (str).
+        Defaults to None.
+    :type callback: function
+
+    :return: The new aggregate hazard with summary.
+    :rtype: QgsVectorLayer
+
+    .. versionadded:: 4.0
+    """
+    impact_keywords = impact.keywords
```
Comment (ismailsunni) [inline safe/gisv4/vector/aggregate_summary.py]: Would it be simpler if we just refer to `impact.keywords` directly? I saw there is only one reading for `impact_keywords` in line 48.

File: safe/definitionsv4/fields.py (hunk 1 above)
Comment (Gustry) [inline safe/definitionsv4/fields.py]: Done in the new commit

File: safe/gisv4/vector/aggregate_summary.py (hunk 2 above)
Comment (Gustry) [inline safe/gisv4/vector/aggregate_summary.py]: Yes of course ;-) I made some change many times and then I deleted some lines

**Pull request merged**